lib/bundler/dsl.rb

```diff
@@ -91,6 +91,25 @@ def gemspec(opts = nil)
       end
     end
 
+    def remove_gems(gems)
```

Comment:
does this need to go in the DSL, or could this be implemented in the Injector class that already handles adding gems?

Reply:
In add command, the injector class appends to a gemfile, whereas in this case, we need to rewrite the whole gemfile. So we will need to put both the logic in `Injector` class.  
We can then create a new method with the same name and move all remove logic to that method(including builder)?

Reply:
this still needs to be moved out of the DSL

Reply:
Moving this method to `Injector` under same name and takes `builder` as input and is a private method?